docs/release-notes/Percona-Everest-1.4.0-(2025-01-07).md

```diff
@@ -70,6 +64,24 @@ To begin your journey with Percona Everest, check out the [Quickstart Guide for
 
     Starting with Percona Everest 1.4.0, we are thrilled to announce that we have added support for PSMDB Operator v1.18.0 and PG operator v2.5.0.
 
+## Google Container Registry (GCR)
+
+!!! warning "GCR deprecation"
+    GCR is set to be **deprecated**, with its official shutdown scheduled for **March 18, 2025**.
+
+    All Percona Everest versions prior to 1.4.0 depend on images hosted on Google Container Registry (GCR). These images will become unavailable after the shutdown date: **March 18, 2025**.
```

Comment:
From the [docs](https://cloud.google.com/artifact-registry/docs/transition/prepare-gcr-shutdown), it looks like reading the images will be available until May 20:
> Effective May 20, 2025, reading images in Container Registry is unavailable.

Reply:
Hi @recharte! Do you think we should change the docs to match the dates?

Reply:
UPD from DS:
we should change the date, bc May 20th is the date that matters for Everest users

Reply:
Thanks @recharte and @oksana-grishchenko

Reply:
Created PR: https://github.com/percona/everest-doc/pull/225 for this.